### PR TITLE
[WIP] Reworking PrairieLearn element docs (close #1050) 

### LINKED ISSUE #1050: HTML Elements Documentation Structure
I'd like to clean up the documentation surrounding HTML elements. 

I'm proposing that:

- [ ] HTML elements are split into subcategories of "decorative" (e.g. `pl_code`, `pl_figure`, ...) and "input fields" (e.g. `pl_number_input`, `pl_multiple_choice`, ... ) 
- [ ] Renaming and tagging test suite questions to match more closely to what they show.
- [ ] Adding links in the documentation to view source of test question code in the repository (e.g. `question.html`, `info.json`, ... )

### PR DESCRIPTION
**Pull request by coatless**
This PR is a WIP. 

There are three goals behind this PR.

1. Reorganize the HTML element documentations into two groups: **Input fields** and **Decorative**
2. Include screenshots of the element in action. 
3. Provide _links_ to: 1. questions using element features and 2. source code on GitHub of the question.

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -140,6 +140,8 @@
   * Change Docker postgresql to do initializations/migrations at build (Dave Mussulman).
 
   * Change the example course to be available to any institution (Matt West).
+  
+  * Change the HTML element documentation to list elements under two categories with example links (James Balamuta).
```
Comment (mwest1066) [inline ChangeLog.md]: Many apologies for letting this PR sit around so long. It needs an update to kebab-case element names and the forthcoming kebab-case change for attribute names (which I think @nwalters512 is planning to push within the next day or so.

Would you mind rebasing this to current master and liaising with @nwalters512 about kebab-case attributes? I promise to merge it fast after this :-)

Comment (coatless) [reply ChangeLog.md]: I'll take care of it later this weekend.

I'm also planning to switch the names of the examples from `elementExamples` to `examplesElement` so they appear together in `questions`.

Comment (mwest1066) [reply ChangeLog.md]: Awesome! And good idea.

Comment (mwest1066) [reply ChangeLog.md]: Maybe you can also check on @mfsilva22's new elements to standardize on naming for these example questions?

**Comment by mwest1066** [comment]
@coatless it looks like this PR never quite got cleaned up for merging. Is it still worth keeping around, or should we just close it out?

**Comment by coatless** [comment]
@mwest1066 let's close this out. I'll end up doing another documentation push later this week as we're heading into Spring Break.

**Pull request closed without merging**